Git/rebase.sublime-syntax

```diff
@@ -0,0 +1,55 @@
+%YAML 1.2
+---
+# Highlight interactive rebase text (git rebase -i).
+
+name: Git Rebase
+file_extensions:
+  - git-rebase-todo
+first_line_match: '^(?:pick|squash|reword|fixup|exec|edit|drop) \h{7,} '
+scope: source.gitrebase
+
+variables:
+  hash: \h{7,}
+
+contexts:
+  prototype:
+    - include: comments
+
+  main:
+    - include: commit
+
+  comments:
+    - match: '#'
+      scope: punctuation.definition.comment.gitrebase
+      push:
+        - meta_scope: comment.line.gitrebase
+        - match: \n
+          pop: true
+        - include: rebase-msg
+
+  # e.g. pick d284bb2 Initial commit
+  commit:
+    - match: ^\s*(squash|reword|pick|fixup|exec|edit|drop|x|s|r|p|f|e|d)\s
```

Comment:
Maybe replace the right-most `\s` with `\b`?

Reply:
If `pick'0123abcd commit text` isn't valid, do you want a `\b` instead of `\s`? This is a case where `\s` makes sense to me.